.github/workflows/pull-request.yml

```diff
@@ -439,8 +445,7 @@ jobs:
       - name: Commit and Push Changes
         run: |
           git config user.name "github-actions[bot]"
-          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
-          git add .
+          git config user.email "${{ github.event.pull_request.user.id }}+${{ github.event.pull_request.user.login }}@users.noreply.github.com"          git add .
```

Comment:
This is going to cause an error. You have the git add at the wrong location
You need to have a `git add --all` to include all new files